app/cdap/components/WrangleHome/Components/WrangleCard/index.tsx

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box, Card, Typography } from '@material-ui/core';
+import { fetchConnectors } from 'components/Connections/Create/reducer';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { getCurrentNamespace } from 'services/NamespaceStore';
+import { BigQuery } from './iconStore/BigQuerySVG';
+import { CloudSQLMySQL } from './iconStore/CloudSQLMySQL';
+import { CloudSQLPostGreSQL } from './iconStore/CloudSQLPostGreSQL';
+import { Database } from './iconStore/Database';
+import { GCS } from './iconStore/GCS';
+import { ImportDatasetIcon } from './iconStore/ImportDatasetIcon';
+import { Kafka } from './iconStore/Kafka';
+import { MySQL } from './iconStore/MySQL';
+import { Oracle } from './iconStore/Oracle';
+import { PostgreSQL } from './iconStore/PostgreSQL';
+import { S3 } from './iconStore/S3';
+import { Spanner } from './iconStore/Spanner';
+import { SQLServer } from './iconStore/SQLServer';
+import { useStyles } from './styles';
+
+const WrangleCard = () => {
+  const [state, setState] = useState({
+    connectorTypes: [],
+  });
+  const getConnectorTypesNames = async () => {
+    let connectorTypes = await fetchConnectors();
```

Comment:
can you add a comment explaining this funciton

Reply:
Done